backend/config/urls.py

```diff
@@ -28,6 +28,12 @@
     url(r"^api/auth/jwt/create_token/", obtain_jwt_token),
     url(r"^api/auth/jwt/refresh_token/", refresh_jwt_token),
     url(r"^api/auth/jwt/verify_token/", verify_jwt_token),
+    url(r"^api/is-test-public", is_test_public_view.TestIsPublic.as_view()),
+    url(r"^api/test-meta-data", test_meta_data_view.TestMetaDataSet.as_view()),
+    url(

```

Comment:
as a note: this second URL is either removed in #334, or will be before I remove the WIP from it